sync15-adapter/examples/sync-pass.rs

```diff
@@ -498,16 +508,21 @@ fn prompt_record_id(e: &PasswordEngine, action: &str) -> Option<String> {
 
 fn main() -> Result<(), failure::Error> {
     env_logger::init();
-    let oauth_data = do_auth(false)?;
 
-    let scope = &oauth_data.keys["https://identity.mozilla.com/apps/oldsync"];
+    let cfg = Config::release()?;
+    let tokenserver_base_url = cfg.token_server_endpoint_url()?.join("../../")?.as_str().to_string();
```

Comment:
Would it help if we exposed tokenserver_base_url instead of token_server_endpoint_url?